telegram/_telegramobject.py

```diff
@@ -243,9 +324,19 @@ def __deepcopy__(self: Tele_co, memodict: dict) -> Tele_co:
         result = cls.__new__(cls)  # create a new instance
         memodict[id(self)] = result  # save the id of the object in the dict
 
-        for k in self._get_attrs_names(
-            include_private=True
-        ):  # now we set the attributes in the deepcopied object
+        setattr(result, "_frozen", False)  # unfreeze the new object for setting the attributes
+
+        # now we set the attributes in the deepcopied object
+        for k in self._get_attrs_names(include_private=True):
+            if k == "_frozen":
+                # Setting the frozen status to True would prevent the attributes from being set
+                continue
+            if k == "api_kwargs":
+                # Need to copy api_kwargs manually, since it's a MappingProxyType is not
+                # pickable and deepcopy uses the pickle interface
+                setattr(result, k, MappingProxyType(deepcopy(dict(self.api_kwargs), memodict)))
+                continue
```

Comment:
hm maybe once again extract this outside the loop?

Reply:
`_get_attrs_names` just returns an iterator now. extracting `_frozen` and `api_kwargs` from that would mean that we'd have to rebuild that iterator afterwards.

Reply:
ah ok, let's leave as is then